Src/ILGPU/Runtime/KernelLoaders.tt

```diff
@@ -552,6 +552,31 @@ namespace ILGPU.Runtime
                 kernelConfig,
                 <#= delegateParams.KernelLaunchArgs #>);
 
+        /// <summary>
+        /// Loads the given kernel and launches it immediately using the given arguments
+        /// with the help of the default accelerator stream.
+        /// </summary>
+        /// <#= delegateParams.TypeParamDocumentation #>
+        /// <param name="action">The action to compile into a kernel.</param>
+        /// <param name="kernelConfig">The launch configuration.</param>
+        /// <#= delegateParams.KernelLaunchParamsDocumentation #>
+        /// <remarks>
+        /// Uses the internal launcher cache that explicitly caches all launched kernels
+        /// using strong references. To avoid caching use the context flag
+        /// <see cref="ContextFlags.DisableKernelLaunchCaching" />.
+        /// </remarks>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public void Launch<<#= delegateParams.TypeParams #>>(
```

Comment:
NOTE: This launcher does not follow the naming convention of the `LoadXXX` methods - LoadXXXStreamKernel indicated using the default stream.

Reply:
As discussed in the `talk-to-dev` session, we will keep the `Launch` to help users launch kernels with the help of default streams.